ProxyAppsSuite/tests/AMG.yaml

```diff
@@ -1,92 +1,58 @@
-_base:
-
-    summary: A test of amg
-
+amg:
+    summary: P1. Laplace problem on 3D cube with 27 point stencil. P2. Non-linear time-dependent problem.
+    variables:
+        problem: 
+            - { name: "problem1", nxval: '100', nyval: '100', nzval: '100', pxval: '2', pyval: '2', pzval: '1', flags: '-problem 1' }
+            - { name: "problem2", nxval: '20', nyval: '20', nzval: '20', pxval: '2', pyval: '2', pzval: '1', flags: '-problem 2' }
```

Comment:
```yaml
            - { name: "problem2", nxval: '100', nyval: '100', nzval: '100', pxval: '4', pyval: '3', pzval: '1', flags: '-problem 2' }
```
This will run a larger problem size, which better exercises the capability of the memory subsystem on the node.